README.md

```diff
@@ -24,6 +24,15 @@ Click to watch it on YouTube:
 
 **Installation**
 
+```
+helm repo add keptn-lifecycle-toolkit https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn-lifecycle-toolkit keptn/lifecycle-toolkit -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+to install the latest release of the Lifecycle Toolkit.
+
+For installing the Lifecycle Toolkit via manifests use:
```

Comment:
```suggestion
```
Use the following command sequence
to install the latest release of the Keptn Lifecycle Toolkit:

helm repo add keptn-lifecycle-toolkit https://charts.lifecycle.keptn.sh
helm repo update
helm upgrade --install keptn-lifecycle-toolkit keptn/lifecycle-toolkit -n keptn-lifecycle-toolkit-system --create-namespace --wait
```


To install the Lifecycle Toolkit via manifests, use:
```